### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: src/ecdsa/keys.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
         ``str`` (only on python2), ``bytes``, ``bytesarray``, ``array.array`
         and ``memoryview`` of those objects.
         Please note that ``array.array` serialisation (converting it to byte
-        string) is endianess dependant! Signature computed over ``array.array``
+        string) is endianness dependent! Signature computed over ``array.array``
```
Comment (tomato42) [inline src/ecdsa/keys.py:64]:
```
$ flake8 setup.py speed.py src
src/ecdsa/keys.py:64:80: E501 line too long (80 > 79 characters)
```

**Review by tomato42 — CHANGES_REQUESTED** [review]
Looks good overall, thanks!
One nit though: it makes one line in the `keys.py` go over the limit, could you break that line?

**Comment by kianmeng** [comment]
> Looks good overall, thanks! One nit though: it makes one line in the `keys.py` go over the limit, could you break that line?

@tomato42 Done. Please check again.

```
$ flake8 setup.py speed.py src
```

**Comment by tomato42** [comment]
Thank you!

**Pull request merged**